features/importing/archivist.feature

```diff
@@ -78,10 +78,9 @@ Feature: Archivist bulk imports
   Then I should see import confirmation
     And I should see "Name change"
   When the system processes jobs
+  # Importer assumes dreamwidth email for works from there
   Then 1 email should be delivered to "jennyst@dreamwidth.org"
```

Comment:
In an ideal world, we'd have an AD&T account on both Dreamwidth and LiveJournal for testing our LJ and DW import code, but for the world we live in, this is fine.